boa/src/syntax/parser/statement/throw/mod.rs

```diff
@@ -49,6 +49,6 @@ impl TokenParser for ThrowStatement {
             }
         }
 
-        Ok(Node::throw(expr))
+        Ok(Throw::new(expr).into())
```

Comment:
This should just return a `Throw`, not a `Node`.

Reply:
Fixed, along with similar instances for other types.